### ISSUE TITLE
titles of special topics courses 

### ISSUE THREAD

**Issue by jk11235** [hyperschedule/hyperschedule#175]
Looks like it doesn't pick up the semester-specific titles for special topics courses, but does pick up the semester-specific descriptions. Look for example at Phil100E CM-1 and CM-2. The titles this semester should be and are in portal: 

> Alienation: Hegel to Foucault. 

The titles should not be 

> Special Topics: Hist of Philos

**Comment by kwshi** [comment]
Thanks for catching this!  Sounds like a problem w/ our data source--I'm looking into it now.

**Comment by kwshi** [comment]
Ok, just poked around a bit.  Indeed, the title on Hyperschedule doesn't exactly match up w/ the one on Portal (and I'll try to find out what that is & try to fix it), it's not incorrect, is it?

On Portal, the course is listed (for me) as "Special Topics: Hist of Philos - Alienation: Hegel to Foucault"
![image](https://user-images.githubusercontent.com/3887189/114608406-82c2fc00-9c52-11eb-8fcb-5c2cedd21bf6.png)
so it seems like the problem is just that we're only picking up half the title instead of the full one.

I'll have to dig a little harder to figure out how to deal w/ that exactly.

**Comment by jk11235** [comment]
Yes, that's right. I was just thinking that, if you pick up only half of the title, it would be better to get the half that specifically describes the course offered Fall21. Thanks!

**Comment by kwshi** [comment]
OK cool.  It's not an easy fix atm (we're getting our data not directly from Portal, and that data simply contains only the first half of the title), but I'll chat w/ the person providing our data to see if we can do anything about this.

**Comment by jk11235** [comment]
Not the biggest deal in the world either way -- just thought you'd like to know. Pretty great that it does pick up the right description for the specific semester.
